openbas-api/src/main/java/io/openbas/utils/challenge/ChallengeAttemptUtils.java

```diff
@@ -0,0 +1,30 @@
+package io.openbas.utils.challenge;
+
+import io.openbas.database.model.ChallengeAttempt;
+import io.openbas.database.model.ChallengeAttemptId;
+import jakarta.validation.constraints.NotBlank;
+import org.jetbrains.annotations.NotNull;
+
+public class ChallengeAttemptUtils {
+
+  private ChallengeAttemptUtils() {}
+
+  public static ChallengeAttemptId challengeAttemptId(
```

Comment:
nitpick: rename to buildChallengeAttemptID or something like that?

Reply:
Fix